### ISSUE TITLE
Mesolve crashes if no `exp_ops` is passed

### ISSUE THREAD

**Issue by abocquet** [PierreGuilmin/torchqdynamics#59]
```
AttributeError                            Traceback (most recent call last)
Cell In[12], line 1
----> 1 tq.mesolve(
      2     H,
      3     [tq.from_qutip(Lb)],
      4     tq.from_qutip(rho),
      5     times,
      6     solver=tq.solver.Rouchon2(dt=times[1])
      7 )

File ~/Programmation/tomcat/qutip-simulations/data-fit/2DC-0004-01-D_run04/../../../../torchqdynamics/torchqdynamics/mesolve/mesolve.py:139, in mesolve(H, jump_ops, rho0, t_save, save_states, exp_ops, solver, gradient_alg, parameters)
    137 if rho0.dim() == 2:
    138     rho_save = rho_save.squeeze(1)
--> 139     exp_save = exp_save.squeeze(1)
    140 if H.dim() == 2:
    141     rho_save = rho_save.squeeze(0)

AttributeError: 'NoneType' object has no attribute 'squeeze'

```

**Comment by PierreGuilmin** [comment]
This is actually fixed in #25 (with a different solution than #60). 😉  Sorry I should have made separated PRs with better namings.

**Comment by PierreGuilmin** [comment]
Yep, because #25 has not been merged yet!

**Comment by PierreGuilmin** [comment]
Sorry it's not #25 😅 It's #54.

**Comment by abocquet** [comment]
Ok I can buy this one, I close #60 and we can close this issue on merging #54

**Comment by PierreGuilmin** [comment]
Fixed by #54.
